src/bin.js

```diff
@@ -58,7 +58,9 @@ Typr._bin = {
 		}
 		return s;
 	},
-	_tdec : typeof window !== 'undefined' && window["TextDecoder"] ? new window["TextDecoder"]() : null,
+	_tdec : function() {

```

Comment:
You've changed the type of `_tdec` from `TextDecoder | null` to a function, so this would certainly break...?

I think maybe a better approach would be to check for `TextDecoder` global directly?

`_tdec : typeof TextDecoder !== 'undefined' ? new TextDecoder() : null`

There's probably also some adjustment to be made in troika-three-text where it [tries to hack in worker support](https://github.com/protectwise/troika/blob/301c34c3341b83eb95afc308bd796bcccb2f5ccd/packages/troika-three-text/rollup.config.build-typr.js#L24-L27).

Reply:
Thank you for your reply.

I'm gonna open an issue in the Drei repos.